src/engine/source/builder/builders/stageBuilderCheck.cpp

```diff
@@ -68,160 +69,63 @@ base::Expression stageBuilderCheckList(const std::any& definition,
     return expression;
 }
 
-base::Expression stageBuilderCheckExpression(const std::any& definition,
-                                             std::shared_ptr<defs::IDefinitions> definitions,
-                                             std::shared_ptr<Registry<Builder>> registry)
+using TermBuilder = std::function<std::function<bool(base::Event)>(const BuildToken&)>;
```

Comment:
Is it correct to use this using for the return of the function?

Reply:
done